front/src/modules/trainschedule/components/ManageTrainSchedule/types.ts

```diff
@@ -69,4 +69,4 @@ export type ValidConfig = {
   speedLimitByTag?: string;
 };
 
-export type FeatureInfoClick = { displayPopup: boolean; feature?: Feature; coordinates?: number[] };
+export type FeatureInfoClick = { feature?: Feature; coordinates?: number[] };
```

Comment:
You can remove the ? :wink: 
```suggestion
export type FeatureInfoClick = { feature: Feature; coordinates: number[] };
```

Reply:
Done